packages/blockbook/src/models.ts

```diff
@@ -257,8 +257,8 @@ export interface Utxo {
   value: string
   height?: number
   confirmations: number
-  address?: string
-  path?: string
+  address: string
```

Comment:
These types are modeled directly off the blockbook types and should stay optional